src/main/java/nl/knaw/dans/lib/util/VersionProvider.java

```diff
@@ -22,14 +22,8 @@
 import java.nio.charset.StandardCharsets;
 
 public class VersionProvider {
-    public String getVersion() throws IOException {
-        try (InputStream inputStream = getClass().getResourceAsStream("/version.txt")) {
-            if (inputStream != null) {
-                return IOUtils.toString(inputStream, StandardCharsets.UTF_8);
-            }
-            else {
-                throw new IllegalStateException("Version file not found");
-            }
-        }
+    public String getVersion(){
```

Comment:
I would expect this to read the manifest of dans-java-utils.jar and not of the main jar of the application. I could be wrong of course.

Reply:
I just arrived at the same conclusion.